server/src-lib/Hasura/Server/Init.hs

```diff
@@ -180,13 +180,20 @@ mkServeOptions rso = do
                                 then WS.PermessageDeflateCompression WS.defaultPermessageDeflate
                                 else WS.NoCompression
                           }
+  webSocketKeepAlive <- KeepAliveDelay . Seconds . secondsToDiffTime . fromIntegral . fromMaybe 5
+      <$> withEnv (rsoWebSocketKeepAlive rso) (fst webSocketKeepAliveEnv)
+
+    -- maybe
+    --   (Just . KeepAliveDelay $ Seconds (secondsToDiffTime 5)) -- default keep-alive interval is 5 seconds
+    --   (Just . KeepAliveDelay . Seconds . secondsToDiffTime)
+    --   <$> (withEnv (rsoWebSocketKeepAlive rso) (fst webSocketKeepAliveEnv))
```

Comment:
Yeah, after a second inspection, I do think the canonical way to get a `Seconds` object is to use the `Num` instance.

```suggestion
  webSocketKeepAlive <- KeepAliveDelay . fromInteger . fromMaybe 5
      <$> withEnv (rsoWebSocketKeepAlive rso) (fst webSocketKeepAliveEnv)
```

Reply:
(Please also implement the `fromIntegral` part of this suggestion.)

Reply:
@abooij what did I forget to do here?

Reply:
@v0d1ch Please avoid the explicit construction of a `Seconds` object using `secondsToDiffTime`, as in my code suggestion above.